pages/talks.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import TalksPage from '../components/talksPage/talksPage';
+import Layout from '../components/layout/layout';
+
+export default function Vast() {
+  return <Layout descriptionContent={<TalksPage />}><h1>talks</h1></Layout>;
```

Comment:
Doesn't the TalksPage component should be a child of the layout and not the prop for the description?

Reply:
where can this <h1> be seen?